README.md

```diff
@@ -73,21 +73,21 @@ With the PROTECTED setting you can define routes that require certain scopes for
 # Require 'employee' scope for read access to /private route
 # Require 'admin' scope for write access to /private route
 'PROTECTED': [
-  ('/private', ['GET', 'HEAD'], 'employee')
-  ('/private', ['POST', 'PUT', 'PATCH', 'DELETE'])
+  ('/private', ['GET', 'HEAD'], ['employee'])
+  ('/private', ['POST', 'PUT', 'PATCH', 'DELETE'], ['admin'])
```

Comment:
Gegeven de functie `convert_scope()` moeten `employee` en `admin` altijd in uppercase right?

Reply:
Als het token van Keycloak komt wel ja..

Reply:
Ik heb het aangepast.